package.json

```diff
@@ -41,60 +45,71 @@
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.7.0",
+    "@embroider/test-setup": "^1.6.0",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "@types/ember": "^3.1.0",
-    "@types/ember-qunit": "^3.4.6",
-    "@types/ember-test-helpers": "^1.0.5",
-    "@types/ember-testing-helpers": "^0.0.3",
-    "@types/ember__test-helpers": "^0.7.8",
+    "@types/ember-qunit": "^5.0.2",
+    "@types/ember-test-helpers": "^1.0.11",
+    "@types/ember-testing-helpers": "^0.0.4",
+    "@types/ember__test-helpers": "^2.8.2",
     "@types/qunit": "^2.9.0",
     "@types/rsvp": "^4.0.3",
     "@typescript-eslint/eslint-plugin": "^2.27.0",
-    "@typescript-eslint/parser": "^2.27.0",
+    "@typescript-eslint/parser": "^5.42.1",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.11.0",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-auto-import": "^2.4.3",
+    "ember-cli": "~4.4.1",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
+    "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript-blueprints": "^2.0.0",
-    "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^2.0.0",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.6.0",
-    "ember-resolver": "^8.0.0",
-    "ember-source": "~3.16.0",
-    "ember-source-channel-url": "^2.0.1",
-    "ember-try": "^1.0.0",
+    "ember-load-initializers": "^2.1.2",
+    "ember-page-title": "^7.0.0",
+    "ember-qunit": "^5.1.5",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "~4.4.0",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^4.18.0",
+    "ember-try": "^2.0.0",
     "ember-welcome-page": "^4.0.0",
-    "eslint": "^6.1.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-ember": "^8.1.1",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^10.6.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-qunit": "^7.2.0",
     "husky": "^4.2.1",
     "lint-staged": "^10.0.2",
     "loader.js": "^4.7.0",
-    "prettier": "^1.18.2",
-    "qunit-dom": "^0.9.2",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.6.2",
+    "qunit": "^2.19.1",
+    "qunit-dom": "^2.0.0",
     "release-it": "^12.2.1",
     "release-it-lerna-changelog": "^1.0.3",
-    "typescript": "^3.5.3"
+    "typescript": "^4.8.4",
+    "webpack": "^5.72.1"
   },
   "engines": {
-    "node": "10.* || >= 12.*"
+    "node": "12.* || 14.* || >= 16"
```

Comment:
Please update to `14.* || 16.* || >= 18` (node 12 is already EOL'ed)

Reply:
fixed in 9b060a19a1e4698862945f731c2a3f257233ab9a